fuzz/Cargo.toml

```diff
@@ -11,6 +11,9 @@ cargo-fuzz = true
 [dependencies]
 libfuzzer-sys = { version = "0.3"}
 fuzz_helper = { path = "fuzz_helper" }
+# We import to explicitly lock the version.
+# Remove this dependency once CTAP is a library.
+bumpalo = "=3.8.0"
```

Comment:
`bumpalo` is actually only needed by the crypto library (because of `ring`). So let's move this line there.

Reply:
Or let's cherry-pick https://github.com/google/OpenSK/pull/534 instead.